### PR TITLE
Elasticsearch storage module wip

### PR DESCRIPTION
**Pull request by Plesoun**

### CODE REVIEW

**Comment by Plesoun** [comment]
@ateska please see, need to refactor, but required functionality is there

**Comment by Plesoun** [comment]
@ateska Please see.

File: asab/storage/elasticsearch.py [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+		Custom ElasticSearch method
+		'''
+		url = "{}_cat/indices/{}?format=json".format(self.ESURL, search_string)
+		async with self.session().request(method="GET", url=url) as resp:
+			assert resp.status == 200, "Unexpected response code: {}".format(resp.status)
+			return await resp.json()
+
+	async def empty_index(self, index):
+		'''
+		Custom ElasticSearch method
```
Comment (ateska) [inline asab/storage/elasticsearch.py:110]: What is "empty_index" ... name of the method should contain **clear** noun -> is is "clear_index" or "create_index" ?

Comment (Plesoun) [reply asab/storage/elasticsearch.py:110]: Right, it is create index, will rename

**Pull request merged**